tools/deploy_to_npm.sh

```diff
@@ -4,14 +4,21 @@ set -e
 
 echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
 
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "$SCRIPT_DIR"/..
+
 # FIXME: since we release from dist directory, README file needs to be copied before release
 cp src/js/README.md dist/
+cp src/js/package.json dist/
 
 cd dist/
 
 PACKAGE_NAME=$(node -p "require('./package.json').name")
 JS_VERSION=$(node -p "require('./package.json').version")
-if [[ ${JS_VERSION} =~ [alpha|beta|rc|dev] ]]; then
+if [[ -n "${DRY_RUN}" ]]; then
```

Comment:
So how would one use it? Does it need to be specified in CI somewhere?

Reply:
I used this manually for this PR `DRY_RUN=1 ./tools/deploy_to_npm.sh`, but I think we should add this to CI so we can check it everytime.

Reply:
I added this to `test-js` CI job. How does it look now?